app/sketched-components/leva-panel/bottom-panel/styles.module.css

```
@@ -0,0 +1,42 @@
+
+    @media (min-width: 728px) {
+      max-width: 410px;
+    }
+  }
+}
+
+.closed {
+  transform: translateY(calc(100% + 2rem));
+}
+
+.noNumberInput :global(input[leva-type="number"]) {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)